docs/guidelines-for-reviewing-datascience-code.md

```diff
@@ -0,0 +1,15 @@
+# Guidelines for Reviewing Data Science Code
+
+Code reviews have been common practice in the software engineering field for some time. They provide developers with a well defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, quality assurance of code and standardize the engineering process across both the code base and the team. As members of the AI CoE in the office of the CTO, we believe that having a well defined approach for how to do the same with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process. Below are the 11 guidelines we use in our group to review data science work.     
+
+1. **Understand the PR** : When asked for a review, make sure to first understand the issue linked and the goal of the pull request.
+2. **NB Review** : Use tools like NB review when possible while reviewing Jupyter Notebooks, this will make reviewing the notebooks easier.
```

Comment:
Can you add a space between each of these numbered sections? It looks a little crowded on the markdown.

Reply:
Sure. I have added some space @MichaelClifford :+1: